app/public/cantusdata/views/folio_set.py

```diff
@@ -17,8 +17,10 @@
 
 
 class ManuscriptFolioSetView(APIView):
-    serializer_class = SearchSerializer
-    renderer_classes = (JSONRenderer,)
+    renderer_classes = (
```

Comment:
Here we define the available renderers with this view. So whereas previously we only `JSONRenderer`, behind the scenes, django rest framework will look at incoming requests and try to determine which of these two renderers should be used for the request.